README.md

```diff
@@ -1,6 +1,8 @@
-# Academy Resident Information API
+# LBH Academy Resident Information API
 
-Base API is a boilerplate code for being reused for new APIs for LBH
+Academy Resident Information API retrieves data from the Academy application, this includes both Claimants and council tax details
+
+The data can be retrieve through the use of a GET endpoint which would retrieve all the data or through a specified ID.
```

Comment:
I think the first retrieve in this sentence should be "retrieved".

Reply:
```suggestion
The data can be retrieved through the use of a GET endpoint which would retrieve all the data or through a specified ID.
```